scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua

```diff
@@ -1,57 +1,137 @@
------------------------------------
+-----------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/keyitems");
+require("scripts/globals/quests");
+require("scripts/zones/RuLude_Gardens/TextIDs");
 
-require("scripts/globals/settings");
-require("scripts/globals/keyitems");
-require("scripts/zones/RuLude_Gardens/TextIDs");
-
 -----------------------------------
 -- onTrade Action
------------------------------------
+-----------------------------------
 
 function onTrade(player,npc,trade)
-end;
+    local meritCount = player:getMeritCount();
+    if (trade:hasItemQty(1127,5) == true and trade:getGil() == 0 and trade:getItemCount() == 5 and meritCount > 2) then
+        if (player:getQuestStatus(JEUNO,NEW_WORLDS_AWAIT) == QUEST_ACCEPTED) then
+            player:startEvent(10135);
+        end
+    elseif (trade:hasItemQty(2955,5) == true and trade:getGil() == 0 and trade:getItemCount() == 5 and meritCount > 3) then
+        if (player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_ACCEPTED) then
+            player:startEvent(10136);
+        end
+    elseif (trade:hasItemQty(2955,10) == true and trade:getGil() == 0 and trade:getItemCount() == 10 and meritCount > 4) then
+        if (player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_ACCEPTED) then
+            player:startEvent(10137);
+        end
+    elseif (trade:hasItemQty(2955,1) == true and trade:hasItemQty(503,1) == true and trade:getGil() == 0 and trade:getItemCount() == 2 and meritCount > 9) then
+        if (player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_ACCEPTED) then
+            player:startEvent(10138);
+        end
+    end
+end;
 
 -----------------------------------
 -- onTrigger Action
 -----------------------------------
 
-function onTrigger(player,npc)
-
-	if (player:hasKeyItem(LIMIT_BREAKER) == false and player:getMainLvl() >= 75) then
-		player:startEvent(0x273D,0x4B,0x02,0x0A,0x07,0x1E,0x049F2F,0x0FFF);
-	else
-		player:startEvent(0x005E);
-	end
-
+function onTrigger(player,npc)
+    if (player:hasKeyItem(LIMIT_BREAKER) == false and player:getMainLvl() >= 75) then
+        player:startEvent(10045,75,2,10,7,30,302895,4095);
+    elseif (player:getMainLvl() == 75 and player:levelCap() == 75 and MAX_LEVEL >= 80 and player:getQuestStatus(JEUNO,NEW_WORLDS_AWAIT) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,1,0);
+    elseif (player:getMainLvl() >= 76 and player:levelCap() == 80 and MAX_LEVEL >= 85 and player:getQuestStatus(JEUNO,EXPANDING_HORIZONS) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,2,0);
+    elseif (player:getMainLvl() >= 81 and player:levelCap() == 85 and MAX_LEVEL >= 90 and player:getQuestStatus(JEUNO,BEYOND_THE_STARS) == QUEST_AVAILABLE) then
+        player:startEvent(10045,0,1,3,0);
+    elseif (player:getMainLvl() >= 91 and player:levelCap() == 90 and MAX_LEVEL >= 95 and player:getQuestStatus(JEUNO,DORMANT_POWERS_DISLODGED) == QUEST_AVAILABLE) then
```

Comment:
haha. http://ffxiclopedia.wikia.com/wiki/Dormant_Powers_Dislodged

The problem is: I can do change anything. You have access for the modification.